README.md

```diff
@@ -165,9 +182,15 @@ $ mix eqc.install --mini
 
 <img src="https://user-images.githubusercontent.com/1253659/84641850-3f163d80-af2e-11ea-98a2-cfb854180222.png" height="80">
 
-
 ## Acknowledgements
 
 Many thanks to [gpb](https://github.com/tomas-abrahamsson/gpb) and
 [golang/protobuf](https://github.com/golang/protobuf) as good examples of
 writing Protobuf decoder/encoder.
+
+## Copyright and License
+
+Copyright (c) 2017 Bing Han
+
+This library is MIT licensed. See the
+[LICENSE](https://github.com/tony612/protobuf-elixir/blob/master/LICENSE) for details.
```

Comment:
Did @tony612 request this change? Let's drop it otherwise.

Reply:
> Did @tony612 request this change? Let's drop it otherwise.

Nope and I'll remove it for now. But good to put copyright and license details in the readme. 

Our of curiosity, looking at the GitHub repo URL change, is this Elixir library now under the co-maintenance between @tony612 and Elixir core team?